runtimevar/filevar/example_test.go

```diff
@@ -24,52 +24,49 @@ import (
 	"gocloud.dev/runtimevar/filevar"
 )
 
-// MyConfig is a sample configuration struct.
-type MyConfig struct {
-	Server string
-	Port   int
-}
-
 func ExampleOpenVariable() {
 	// Create a temporary file to hold our config.
 	f, err := ioutil.TempFile("", "")
 	if err != nil {
 		log.Fatal(err)
 	}
-	if _, err := f.Write([]byte(`{"Server": "foo.com", "Port": 80}`)); err != nil {
+	if _, err := f.Write([]byte("hello world")); err != nil {
 		log.Fatal(err)
 	}
 
-	// Create a decoder for decoding JSON strings into MyConfig.
-	decoder := runtimevar.NewDecoder(MyConfig{}, runtimevar.JSONDecode)
-
 	// Construct a *runtimevar.Variable pointing at f.
-	v, err := filevar.OpenVariable(f.Name(), decoder, nil)
+	v, err := filevar.OpenVariable(f.Name(), runtimevar.StringDecoder, nil)
 	if err != nil {
 		log.Fatal(err)
 	}
 	defer v.Close()
+	// Ignore unused variables in example:
```

Comment:
Not really ignoring here either, are you?

Reply:
Same.